docs/src/APIs/shared_utilities.md

```diff
@@ -55,7 +55,7 @@ ClimaLand.source!
 ClimaLand.AbstractBoundary
 ClimaLand.TopBoundary
 ClimaLand.BottomBoundary
-ClimaLand.boundary_flux
+ClimaLand.boundary_flux!
```

Comment:
Just pointing out that you are modifying the declared public API of the package, so this would be a breaking change if we wanted to be pedantic.

Reply:
I added a breaking change to NEWS.md